### DISCUSSION TITLE
PIXART-α : First Open Source Rival to Midjourney - Better Than Stable Diffusion SDXL - Full Tutorial

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [PixArt-alpha/PixArt-alpha#52] [Show and tell]
[**PIXART-α : First Open Source Rival to Midjourney - Better Than Stable Diffusion SDXL - Full Tutorial**](https://youtu.be/ZiUXf_idIR4)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/NccIesCaYHEpKshYB-jF0.png)](https://youtu.be/ZiUXf_idIR4)

**Comment by NonaBuhtig** [comment]
You shouldn't be allowed here, This video is exclusive to paying patreon users and there isn't any links to the Pixart-alpha GitHub repository.
Shame on you for spamming all these great projects with your advertising !

**Reply by FurkanGozukara** [reply]
you didn't even watch it right? it contains 1 week of work and whole amount of information. what have you provided to people?

**Reply by NonaBuhtig** [reply]
I see the video ! nothing relevant for non patreon viewer ... Even the link to this github is missing. what I provide to people is not your concern and do not need a credit card. Thanks.

**Reply by FurkanGozukara** [reply]
nope you didn't watch it. it provides so much information for non patreon supporters as well. it is literally first video advertising PixArt to the world.

**Reply by mrT333** [reply]
Without this video i wouldn't have come across Pixart even though I am an avid Stable Diffusion power user. I didn't care about the installation part as I am a software engineer and can figure that out by myself.

For me the value was in the comparison with SDXL regarding the adherence to the prompt that was shown in the video. It made me curious enough to check out the Pixart github and that is where I stumbled upon the absurdly high training efficiency which finally sold me on Pixart as that is how it could surpass even the best stable diffusion finetunes once the community starts adopting and training on Pixart.

I am sure there are much more people than me who stumbled upon Pixart due to this video but remain silent lurkers which make it i.m.o. a valuable contribution to the Pixart community with or without some added paywalled scripts.

Although I would appreciate a direct reference to the Pixart github in the video description @FurkanGozukara as I imagine there might be a lot of people who miss out on the technical marvel that is Pixart just because they didn't bother googling it but would have clicked on a url if referenced in the description :)

**Reply by FurkanGozukara** [reply]
@mrT333 ty so much for comment. I added repo link to the description of the video so people can click :)

**Reply by NonaBuhtig** [reply]
@mrT333, you are speaking for yourself and I'm glad that you have the necessary skills.

I'm not here to fight against him or against his fanboys. I only want to spot that this is an open source research. At minima an "how to" for people that don't have access to his "installer" batch files would have been welcome before posting his commercial here.

FurkanGozukara by spamming every github repo, reddit, hugginface with his videos is building a solid community of followers.
But adding more visibilty to a research does not excuse the fact that he doesn't care about people that aren't patreon member. And because you have some skills that a lots of people don't have you are supporting his "closed" paywall behavior.

You seems to forget that Stable-diffusion and all research around is free and open source. making money with this is a part of the deal. But helping people who don't have knowledge and/or money is also.

**Reply by mrT333** [reply]
@NonaBuhtig my point is not that the needs of people who can't code don't matter. My point is rather that there are people with all different kinds of backgrounds that have different needs and therefore value information differently.

While I acknowledge that your need for understanding the installation process might not be met through this video it is just not right to say that the whole video doesn't provide any value to anybody when in fact it does, just to another segment of the audience with different needs than yours. 

There might be other videos where I don't feel satisfied because the information contained isn't helping me. But if most of it is helping other segments of the audience I wouldn't try shaming the creators for sharing their hard work even if "just" 90% of it is given for free and they provide an additional 10% that is paywalled to make the effort pay for itself. (And even those 10% - if we're honest - is not critical to the installation but rather optional as you can search for the Pixart repo and just follow the instructions there.)

I understand that from your perspective it might look like promotion without nothing of value given and everything being paywalled. I felt compelled to comment to show that this view is certainly not valid for everybody, taking myself as an example as my experience was the exact opposite: everything I cared about was freely available and made me notice Pixart in the first place thereby acting as a free promotion **for** Pixart giving value to the community rather than taking away from it.

TL/DR: Just because something doesn't provide value to every segment of the audience doesn't mean it has no value for others. 

As the saying goes: "You can please some of the people all of the time, you can please all of the people some of the time, but you can't please all of the people all of the time."

**Reply by NonaBuhtig** [reply]
@mrT333,
I understand what you're saying and I agree. I was certainly too harsh in my opinion. Unfortunately, my annoyance got the better of me.

**Reply by mrT333** [reply]
> @mrT333, I understand what you're saying and I agree. I was certainly too harsh in my opinion. Unfortunately, my annoyance got the better of me.

I think highly of people with a great degree of self reflection and hope you get the value that you seek as soon as the videos from other AI-influencers on Pixart follow in a few days (probably).

I wouldn't be surprised if the above video pops up in the yt feed of the other AI-Youtubers and they suddenly get "inspired" to do their own review of Pixart with their own installation instructions ...as it is usually the case in that group :)
